sensors/nxt_i2c_sensor_defs.c

```diff
@@ -2044,6 +2044,83 @@ const struct nxt_i2c_sensor_info nxt_i2c_sensor_defs[] = {
 			},
 		}
 	},
+	[MS_IR_THERMOMETER] = {
+		/**
+		 * @vendor_name: mindsensors.com
+		 * @vendor_part_number: IRThermometer
+		 * @vendor_part_name: IRThermometer
```

Comment:
Let's use the same name as on the mindsensors.com website. `IR Temperature Sensor for EV3 or NXT`